docs/howto/release-git-lfs.md

```diff
@@ -155,18 +162,16 @@ equal to 0, we say that we are releasing a MINOR version of Git LFS, in the
      similar to the following:
 
      ```diff
-     diff --git a/_config.yml b/_config.yml
-     index 03f23d8..6767f6f 100644
-     --- a/_config.yml
-     +++ b/_config.yml
+     --- _config.yml
+     +++ _config.yml
      @@ -1,7 +1,7 @@
       # Site settings
       title: "Git Large File Storage"
       description: "Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com or GitHub Enterprise."
-     -git-lfs-release: 2.5.1
-     +git-lfs-release: 2.5.2
+     -git-lfs-release: M.(N-1).0
+     +git-lfs-release: M.N.0
```

Comment:
Super minor nit: Maybe replace the first `0` with a `P`? Because we usually do not release a new minor version without any patch releases?!